DashKit/ReservoirLevel.swift

```diff
@@ -14,14 +14,16 @@ public enum ReservoirLevel: RawRepresentable, Equatable {
     public typealias RawValue = Int
 
     case valid(Double)
-    case aboveThreshold
+    case aboveThreshold // the threshold is 50 units remaining
 
     public func asPercentage() -> Double? {
         switch self {
         case .aboveThreshold:
+            // reservoir has more than 50 units remaining
             return nil
         case .valid(let value):
-            return min(1, max(0, value / Pod.reservoirCapacity))
+            // design requested that reservoir level start display at 50%. This means that 50 units remaining = 50%, 25 units = 25% and so on.
```

Comment:
The Omnipod reservoir is 200U.  With this change, the reservoir icon will show 50% when the reservoir is actually 25% full. Is that really the intention here?

Reply:
correct. This was requested from design. I'll put it on my standup topic list to confirm.

Reply:
Would be good to hear the reasoning here. I don't think the iconography needs to match the physical device with complete accuracy, but this showing double the reality.